server/scsynth/SC_Jack.cpp

```diff
@@ -305,6 +305,9 @@ void SC_JackDriver::ConnectClientOutputs(const char* pattern) {
     free(ports);
 }
 
+
+static inline double sc_JACKOSCTimeSeconds() { return (uint64)OSCTime(getTime()) * kOSCtoSecs; }
```

Comment:
1. `inline` is not necessary here
2. `uint64` cast is unnecessary since the result of OSCTime will be implicitly converted to a double to perform the following multiplication
3. please place this at the start of the file with the other free functions (e.g., oscTimeNow, which you could actually use in this function)
4. we use webkit naming conventions:

https://github.com/supercollider/supercollider/wiki/Cpp-code-style-guidelines#naming
https://webkit.org/code-style-guidelines/#names-basic

in particular, note:

> Lower-case the first letter, including all letters in an acronym, in a variable or function name.

so this should be `sc_jackOscTimeSeconds()`.

i'm also dubious about adding the `sc_` prefix, since that is typically (at least AFAICT) reserved for things that are kind of like the "supercollider API". what do you think about removing it?